app/assets/stylesheets/application.css.scss

```diff
@@ -18,3 +18,170 @@
 @import "bootstrap";
 @import "font-awesome-sprockets";
 @import "font-awesome";
+
+body {
+  background-color: #ebe7df;
+  color: #333333;
+  font-family: "century gothic", "Helvetica Neue", Helvetica, Arial, sans-serif;
+  font-size: 14px;
+  line-height: 1.42857143;
+}
+
+.row h1 {
+  background: #418a95;
+  color: #fff;
+  font-weight: bold;
+  font-size: 20;
+  margin-bottom: 0;
+  padding: 30 10;
+}
+
+.table {
+  box-shadow: 0 10 10 rgba(0, 0, 0, 0.2);
+  margin-bottom: 20;
+  max-width: 100%;
+  width: 100%;
+}
+
+.table tr {
+  border-top: 1px solid #ebe7df;
+}
+
+.table thead tr {
+  border-bottom: 1px solid #ebe7df;
+  border-top: 1px solid #63acb7;
+  background: #63acb7;
+  color: #fff;
+}
+
+.table thead tr:first-child {
```

Comment:
Merge rule `.table thead tr:first-child` with rule on line 39<br>Selector should have depth of applicability no greater than 2, but was 3